src/components/Home.js

```diff
@@ -0,0 +1,148 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import { Box, CardActionArea } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import LanguageIcon from '@mui/icons-material/Language';
+import { useNavigate } from 'react-router-dom';
+import { fetchCountry } from '../redux/home/HomeSlice';
+import { fetchCountryDetail, selectedCountryAction } from '../redux/detail/DetailSlice';
+
+function Home() {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.country.status);
+  const countries = useSelector((state) => state.country.countrystore);
+  const searchInput = useSelector((state) => state.country.searchresult);
+  const navigate = useNavigate();
+  useEffect(() => {
```

Comment:
- [ ] Nice done 👍 
![Capture](https://user-images.githubusercontent.com/63972670/225394108-9b139e30-4922-44f9-9de5-dfe5b73e876e.PNG)

Reply:
@OLIPLICHE I already link a video on Pull request is ithttps://www.loom.com/share/393dace41dcd40e2b7c35dae1d592611